include/simulation_time/Simulation_Time.hpp

```diff
@@ -109,6 +127,42 @@ class Simulation_Time
         return current_timestamp;
     }
 
+    inline int next_timestep_index(int epoch_time_seconds)
+    {
+        return int(epoch_time_seconds - start_date_time_epoch) / output_interval_seconds;
+    }
+
+    inline int next_timestep_index()
+    {
+        return next_timestep_index(current_date_time_epoch);
+    }
+
+    inline time_t next_timestep_epoch_time(int epoch_time_seconds){
```

Comment:
`time_t` wasn't previously part of the interface of `Simulation_Time` other than what's passed in through the `simulation_time_params` constructor. Could we use the better specified `std::chrono` types and routines for this instead?

Reply:
Arguments that name a duration with a unit should just be `std::chrono::duration`, too